sway/desktop/output.c

```diff
@@ -268,6 +291,22 @@ static int output_repaint_timer_handler(void *data) {
 			wlr_output_state_set_gamma_lut(&pending, 0, NULL, NULL, NULL);
 		}
 	}
+	
+	if (output->can_tear) {
+		pending.tearing_page_flip = true;
+		
+		if (!wlr_output_test_state(output->wlr_output, &pending)) {
+			sway_log(SWAY_ERROR, "Output test failed on '%s', retrying without tearing page-flip",
```

Comment:
I'd suggest downgrading this to `SWAY_DEBUG`, because we will hit this quite often when the cursor plane is used.

Reply:
Changed.